src/shapes/Polyline.ts

```diff
@@ -323,31 +323,21 @@ export class Polyline<
    * @static
    * @memberOf Polyline
    * @param {SVGElement} element Element to parser
-   * @param {Function} callback callback function invoked after parsing
    * @param {Object} [options] Options object
    */
-  static fromElement(
-    element: SVGElement,
-    callback: (poly: Polyline | null) => any,
-    options?: any
-  ) {
-    if (!element) {
-      return callback(null);
-    }
+  static async fromElement(element: SVGElement, options?: any) {
```

Comment:
```suggestion
  static async fromElement(element: SVGElement, options?: Partial<SerializedPolylineProps>) {
```

Reply:
you know i have NO IDEA what that options are for. Not gonna type till i discover.
Everything comes from the element, 2-3 classes have options, the others have no.
Seems to me options are those:
```js
    options = {
      ...applyViewboxTransform(doc),
      crossOrigin,
      svgUid,
      signal,
    };
```
And they have nothing to do with the partial props of the instance itslef.
I ll figure it out in another PR